.github/workflows/prepare-idl-release.yml

```diff
@@ -0,0 +1,89 @@
+# Create a PR that bumps the version of packages/idl/package.json when changes
+# are detected compared with the latest version of the package published on NPM.
+#
+# Typical weekly schedule:
+# - New browser-specs release on Monday,
+# - Integration of new version of browser-specs in Reffy on Tuesday
+# - Cleanup job on Wednesday
+# - (This job) New @webref/idl package release on Thursday
+
+name: Prepare new @webref/idl release PR if needed
+
+on:
+  schedule:
+    - cron: '0 4 * * 4'
+  workflow_dispatch:
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Setup node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: 14.x
+
+    # Checkout webref and prepare new release in a separate folder so that we
+    # may also install the current version of @webref/idl for comparison.
+    - name: Checkout webref
+      uses: actions/checkout@v2
+      with:
+        path: webref
+
+    - name: Prepare new release
+      run: |
+        npm ci
+        npm run prepare
+      working-directory: webref
+
+    - name: Test new release
+      run: npm test
+      working-directory: webref
+
+    - name: Install current @webref/idl package for comparison
+      run: |
+        mkdir idl
+        cd idl
+        npm install @webref/idl
+
+    - name: Compute diff between new release and current package
+      # For details on syntax to use to export a multiline string to an
+      # environment variable, see:
+      # https://github.com/actions/toolkit/blob/main/docs/commands.md#set-an-environment-variable
+      # The diff does not take the package.json file into account because
+      # "npm install" adds properties that start with "_" to that file which do
+      # not exist in the repo version of the file.
+      run: |
+        echo 'IDL_DIFF<<EOF' >> $GITHUB_ENV
+        diff idl/node_modules/@webref/idl webref/packages/idl --ignore-trailing-space --exclude=package.json >> $GITHUB_ENV
```

Comment:
Is the path here right? With no `working-directory` here, what's the implicit working directory? I would have guessed it's the default location for a checkout, not the parent and this seems to assume.

Reply:
I do not really know but isn't that the same thing as in the `update-ed` job?
https://github.com/w3c/webref/blob/master/.github/workflows/update-ed.yml#L37

Reply:
Perhaps, I didn't compare it to anything. Let's try and see if it works?